docs/topics/cache.txt

```diff
@@ -1277,12 +1277,47 @@ Here are some more examples:
 * ``no_transform=True``
 * ``must_revalidate=True``
 * ``stale_while_revalidate=num_seconds``
+* ``no_cache=True``
 
 The full list of known directives can be found in the `IANA registry`_
 (note that not all of them apply to responses).
 
 .. _IANA registry: https://www.iana.org/assignments/http-cache-directives/http-cache-directives.xhtml
 
+``no_cache`` is a special case in that it can be set to ``True`` or to a list of values.
+
+If you call ``cache_control(no_cache='...')`` multiple times, all the values will used in the
+resulting ``Cache-Control`` header::
+
+    from django.views.decorators.cache import cache_control, patch_cache_control
+
+    @cache_control(no_cache=['Set-Cookie', 'foo'])
```

Comment:
@felixxm I don't like using `foo` and `bar` in docs, but I couldn't find any real-world example of what values people would use, other than `Set-Cookie` as mentioned by the ticket author.

Reply:
IMO, using multiple field names in `no-cache` directive is so niche that we don't need to explain this in our documentation. I'm going to leave only `versionchanged` admonition.